indico/htdocs/js/indico/modules/timetable/timetable/Management.js

```diff
@@ -682,19 +682,19 @@ type("RescheduleDialog", ["ExclusivePopupWithButtons"], {
 type("FitInnerTimetableDialog", ["ConfirmPopup"], {
 
     /**
-     * Returns the title of the timetable/session/entry
+     * Returns the title of the session block
      */
-    __getSessionTitle: function(){
-        return '"' + this.tt.contextInfo.title + '"';
+    __generateSessionBlockTitle: function(slotData) {
```

Comment:
i'm curious, why pass it as an argument instead of accessing it via `this.tt.contextInfo` here?

Reply:
Not really a reason, I've just based it on a similar function but your way is actually cleaner.